packages/rum/README.md

```diff
@@ -133,22 +171,6 @@ init(configuration: {
 })
 ```
 
-### Name click actions
-
-The RUM library uses various strategies to automatically name click actions. If you want more control, define a `data-dd-action-name` attribute on clickable elements (or any of their parents) to name the action, for example:
```

Comment:
is it documented elsewhere?

Reply:
I'll open a PR to add it in the normal doc.